### PR TITLE
RW rounding and better define RW to be viewport width when dimensions unknown

### PR DESCRIPTION
**Pull request by yoavweiss**
Closes https://github.com/igrigorik/http-client-hints/issues/38

Hey Ilya,

Can you take a look?

### CODE REVIEW

File: draft.md [hunk 1]
```diff
@@ -165,7 +165,7 @@ If Content-DPR occurs in a message more than once, the last value overrides all
 
 # The RW Client Hint
 
-The "RW" header field is a number that, in requests, indicates the Resource Width (RW) in CSS px, which is either the display width of the requested resource (e.g. display width of an image), or the layout viewport width if the resource does not have a display width (e.g. a non-image asset).
+The "RW" header field is a number that, in requests, indicates the Resource Width (RW) in CSS px, which is either the rounded up display width of the requested resource (e.g. display width of an image), or the layout viewport width if the resource does not have a display width (e.g. a non-image asset) or its display width is not known.
```
Comment (igrigorik) [inline draft.md]: Rounded up != ceiling. I think we should be more specific.. How about:

The "RW" header field is a number that, in requests, indicates the Resource Width (RW) in CSS px, which is either the rounded up display width of the requested resource (e.g. display width of an image), or the layout viewport width if its display width is either not known at the time of the request or the resource does not have a display width (e.g. a non-image asset). The provided CSS px value is a number rounded to the largest smallest following integer (i.e. ceiling function).

**Comment by yoavweiss** [comment]
Yeah, that leaves less room for misinterpretation.
Corrected according to the comment.

File: draft.md [hunk 2]
```diff
@@ -165,7 +165,9 @@ If Content-DPR occurs in a message more than once, the last value overrides all
 
 # The RW Client Hint
 
-The "RW" header field is a number that, in requests, indicates the Resource Width (RW) in CSS px, which is either the display width of the requested resource (e.g. display width of an image), or the layout viewport width if the resource does not have a display width (e.g. a non-image asset).
+The "RW" header field is a number that, in requests, indicates the Resource Width (RW) in CSS px, which is either the rounded up display width of the requested resource (e.g. display width of an image), 
+or the layout viewport width if its display width is either not known at the time of the request or the resource does not have a display width (e.g. a non-image asset).
+The provided CSS px value is a number rounded to the largest smallest following integer (i.e. ceiling function).
```
Comment (igrigorik) [inline draft.md]: nitpick, can we avoid the linebreaks to keep the formatting consistent?

**Comment by yoavweiss** [comment]
I would have preferred to align the rest of the document, but... :D

**Comment by igrigorik** [comment]
thanks!

**Pull request merged**